### PR TITLE
feature(tiger): Добавит компонент - чекбокс

### PR DESCRIPTION
**Pull request by dontuse**

### CODE REVIEW

File: src/styles/mixins/icons.scss [hunk 1]
```diff
@@ -58,3 +58,8 @@
   $svg-string: encode-base64('<svg fill="#{$color}" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1792 1120q0 166-127 451-3 7-10.5 24t-13.5 30-13 22q-12 17-28 17-15 0-23.5-10t-8.5-25q0-9 2.5-26.5t2.5-23.5q5-68 5-123 0-101-17.5-181t-48.5-138.5-80-101-105.5-69.5-133-42.5-154-21.5-175.5-6h-224v256q0 26-19 45t-45 19-45-19l-512-512q-19-19-19-45t19-45l512-512q19-19 45-19t45 19 19 45v256h224q713 0 875 403 53 134 53 333z"/></svg>');
   background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
 }
+
+@mixin check-icon($color: #1f2532) {
+  $svg-string: encode-base64('<svg fill="#{$color}" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1671 566q0 40-28 68l-724 724-136 136q-28 28-68 28t-68-28l-136-136-362-362q-28-28-28-68t28-68l136-136q28-28 68-28t68 28l294 295 656-657q28-28 68-28t68 28l136 136q28 28 28 68z"/></svg>');
+  background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
```
Comment (apress-hound) [inline src/styles/mixins/icons.scss]: `50%/contain` should be written with a single space on each side of the operator: `50% / contain`

File: src/Checkbox/e-checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import '../styles/mixins/icons';
+
+.e-checkbox {
+  box-sizing: border-box; width: 16px; height: 16px; background: white; border: solid 1px #e0e0e0; border-radius: 2.7px; text-indent: -99999px; cursor: pointer; display: inline-block;
```
Comment (apress-hound) [inline src/Checkbox/e-checkbox.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

File: dist/assets-table.bundle.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "dd512c19154e77df04b2"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "16b1a124959f20459252"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-table.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

File: dist/app.bundle.js [hunk 4]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "dd512c19154e77df04b2"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "16b1a124959f20459252"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/app.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Too many errors. (0% scanned).

**Comment by dontuse** [comment]
@Kupov

File: src/styles/mixins/icons.scss (hunk 1 above)
Comment (apress-hound) [inline src/styles/mixins/icons.scss]: `50%/contain` should be written with a single space on each side of the operator: `50% / contain`

File: src/Checkbox/e-checkbox.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
```
Comment (apress-hound) [inline src/Checkbox/e-checkbox.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

File: dist/assets-table.bundle.js (hunk 3 above)
Comment (apress-hound) [inline dist/assets-table.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

File: src/Checkbox/Checkbox.jsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React, {PropTypes} from 'react';
+import {block} from '../utils';
+import './e-checkbox.scss';
+
+const b = block('e-checkbox');
+
+const Checkbox = props =>
+  <div
+    onClick={() => props.onChange(!props.checked)}
```
Comment (Kupov) [inline src/Checkbox/Checkbox.jsx]: что за сделай чексбокс чексбоксом

Comment (dontuse) [inline src/Checkbox/Checkbox.jsx]: нам он тут не нужен, у нас спа

**Pull request closed without merging**